internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -105,26 +106,29 @@ func (r *AddOnDefinitionResource) Schema(ctx context.Context, req resource.Schem
 				},
 			},
 			"readme_markdown_template": schema.StringAttribute{
-				Description: "The markdown template for the Add-on Definition README.",
+				Description: "The markdown template for the Add-on Definition README. Must be base 64 encoded.",
```

Comment:
We should just base64 encode it for the user. Or at least document that users are expected to set this field using https://developer.hashicorp.com/terraform/language/functions/base64encode for this field. But given it's always required, maybe we should just be base64 encoding it ourselves transparently behind the scenes in the provider.

Reply:
Basically the fact that this field was made to be base64 encoded is tech debt that we are planning to remove, so doing this behind the scenes means this providers resource in users module code won't need to be changed in the future when we remove the base64 field.

Reply:
I am going to defer this change to another seperate PR then because for the time being all the Waypoint providers are doing this in the same way, so we should change all of them at once. I am going to write up another Jira ticket to track that.

Reply:
Actually instead of doing that I am just going to make the change here across the different Waypoint resources.